install.sh

```diff
@@ -15,9 +15,6 @@ if echo $REPLY | grep -E '^[Yy]$' > /dev/null; then
             exit 1
         fi
 
-        # Needed for proper building under macOS
-        xcode-select --install
```

Comment:
We require xcode to be installed prior to executing this, but the command line tools come automatically